src/pyhf/infer/calculators.py

```diff
@@ -125,8 +143,17 @@ def distributions(self, poi_test):
         """
         if self.sqrtqmuA_v is None:
             raise RuntimeError('need to call .teststatistic(poi_test) first')
-        sb_dist = AsymptoticTestStatDistribution(-self.sqrtqmuA_v)
-        b_dist = AsymptoticTestStatDistribution(0.0)
+
+        if self.base_distr == 'normal':
+            cutoff = -1e10
+        elif self.base_distr == 'clipped_normal':
+            cutoff = -self.sqrtqmuA_v
+        else:
+            raise RuntimeError(
```

Comment:
Shouldn't this error be caught at construction (rather than waiting on a call to `distributions`)?

Additionally, maybe `Unsupported` for this error as it's an unsupported base distribution configuration.

Reply:
maybe `ValueError` is enuogh?